{"name": "test_stackify_application_as", "status": "failed", "statusDetails": {"message": "AssertionError: Contents Under Languages are Not Displayed As Expected", "trace": "self = <stackify_application_test.TestStackifyApplicationAS object at 0x10a5ef390>\nreadJson = {'expected_title': 'Application Performance Monitoring Tools For Developers', 'stackify_url': 'https://stackify.com/'}\n\n    def test_stackify_application_as(self, readJson):\n        HomePage = HomePageAS(self.driver)\n        LoginPage = LoginPageAS(self.driver)\n        RubyPage = RubyPageAS(self.driver)\n        HomePage.launch_application(readJson['stackify_url'])\n        HomePage.validate_logo()\n        HomePage.validate_url(readJson['stackify_url'])\n        HomePage.validate_title(readJson['expected_title'])\n        HomePage.verify_header_menu_options()\n>       HomePage.validate_content_under_solutions()\n\ntest_file/stackify_application_test.py:19: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.home_page.HomePageAS object at 0x10a6438c0>\n\n    def validate_content_under_solutions(self):\n        actions = ActionChains(self.driver)\n        Solutions = self.driver.find_element(*locators[\"SOLUTIONS\"])\n        actions.move_to_element(Solutions).perform()\n        time.sleep(3)\n        language_links_expected = [\".NET\", \"Java\", \"PHP\", \"Node.js\", \"Ruby\", \"Python\"]\n        content_displayed = []\n        for i in language_links_expected:\n            print(i)\n            by, value = locators[\"LANGUAGES_CONTENT\"]\n            format_value = value.format(content=i)\n            print(format_value)\n            element_found = self.driver.find_element(by, format_value).is_displayed()\n            content_displayed.append(element_found)\n        if all(content_displayed):\n            print(\"All Contents Under Languages are Displayed As Expected\")\n        else:\n>           assert False, \"Contents Under Languages are Not Displayed As Expected\"\nE           AssertionError: Contents Under Languages are Not Displayed As Expected\n\npages/home_page.py:80: AssertionError"}, "attachments": [{"name": "stdout", "source": "d87464cb-69e4-4fb9-a766-179867533a04-attachment.txt", "type": "text/plain"}], "start": 1733922711466, "stop": 1733922715795, "uuid": "6339d46d-e942-4eb7-944e-0f399ce38ebe", "historyId": "acf25f3de598503211b2c01336ee2304", "testCaseId": "acf25f3de598503211b2c01336ee2304", "fullName": "test_file.stackify_application_test.TestStackifyApplicationAS#test_stackify_application_as", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('browser_cbt')"}, {"name": "parentSuite", "value": "test_file"}, {"name": "suite", "value": "stackify_application_test"}, {"name": "subSuite", "value": "TestStackifyApplicationAS"}, {"name": "host", "value": "Apples-MacBook-Pro.local"}, {"name": "thread", "value": "13747-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_file.stackify_application_test"}]}